checks/containers/container_engine/xccl_tests.py

```diff
@@ -13,28 +13,31 @@
 
 from container_engine import ContainerEngineMixin  # noqa: E402
 
-
-class XCCLTestBase(rfm.RunOnlyRegressionTest, ContainerEngineMixin):
+class XCCLTestsBase(rfm.RunOnlyRegressionTest):
     valid_prog_environs = ['builtin']
     maintainers = ['amadonna', 'VCUE']
     sourcesdir = None
     test_name = parameter(['all_reduce', 'sendrecv'])
     num_nodes = variable(int, value=2)
     min_bytes = variable(str, value='1024M')
     max_bytes = variable(str, value='1024M')
-    container_env_table = {
-        'annotations.com.hooks': {
-            'aws_ofi_nccl.enabled': 'true'
-        }
-    }
+    tags = {'production', 'maintenance'}
     env_vars = {
         'NCCL_DEBUG': 'Info',
+    }
 
-        # Disable MCA components to avoid warnings
-        'PMIX_MCA_psec': '^munge',
-        'PMIX_MCA_gds': '^shmem2'
+    reference_per_test = {
+        'sendrecv': {
+            '*': {
+                'GB/s': (24.0, -0.05, None, 'GB/s')
+            }
+         },
+        'all_reduce': {
+            '*': {
+                'GB/s': (90.0, -0.05, None, 'GB/s')
+            }
+        }
     }
-    tags = {'production', 'ce', 'maintenance'}
```

Comment:
I removed the `ce` tag since this base class is used for both uenv and ce. Is it important to have the `ce` tag for the ce variants of the test?